eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -35,6 +35,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
```

Comment:
That is precisely my understanding. Meaning that the inline `{@inheritDoc}` doesn't serve any purpose. The non-inline version doesn't exist, but is being considered for inclusion.